pkg/services/ngalert/remote/external_alertmanager.go

```diff
@@ -282,9 +283,9 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 }
 
 // TODO: change implementation, this is only useful for testing other methods.
-func (am *externalAlertmanager) postConfig(ctx context.Context, rawConfig string) error {
-	url := strings.TrimSuffix(am.url, "/alertmanager") + "/api/v1/alerts"
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, url, strings.NewReader(rawConfig))
+func (am *ExternalAlertmanager) postConfig(ctx context.Context, rawConfig string) error {
+	alertsURL := strings.TrimSuffix(am.url, "/alertmanager") + "/api/v1/alerts"
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, alertsURL, strings.NewReader(rawConfig))
```

Comment:
Had to rename this URL as I had a warning on: variable conflicts with a package name.